1001-2000/LIT1340EnochE.xml

```diff
@@ -48,30 +50,120 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <p>A TEI file converted from the ETHIO-authority google spreadsheet</p>
          </sourceDesc>
          <sourceDesc>
-               <listBibl type="clavis">
-                  <bibl type="CAVT">
-                     <ptr target="bm:CAVT"/>
-                     <citedRange unit="item">61</citedRange>
-                  </bibl>
-               </listBibl>
+            <listWit rend="edition">
+
+               <witness corresp="BAVet71" xml:id="bavet71">BAV Aeth. 71</witness>
```

Comment:
As far as I understood, the name of the manuscript should be given completely or deleted. 
So, either: `<witness corresp="BAVet71" xml:id="bavet71">Vatican City, Biblioteca Apostolica Vaticana, Aeth. 71</witness>` 
or: `<witness corresp="BAVet71" xml:id="bavet71"/>`

Reply:
exactly. it is always recommended to delete, unless there should be something additional to the shelfmark (folia, date, comment, etc), just as it is with all named entities